gtsfm/utils/io.py

```diff
@@ -151,9 +158,19 @@ def write_images(gtsfm_data: GtsfmData, save_dir: str) -> None:
     """
     os.makedirs(save_dir, exist_ok=True)
 
+    num_imgs = gtsfm_data.number_images()
+    # TODO: compute this
+    mean_obs_per_img = 0
+
+    # TODO: compute this
+    img_fname = "dummy.jpg"
+
     file_path = os.path.join(save_dir, "images.txt")
     with open(file_path, "w") as f:
-        f.write("# Number of cameras: {}\n".format(gtsfm_data.number_images()))
+        f.write("# Image list with two lines of data per image:")
+        f.write("#   IMAGE_ID, QW, QX, QY, QZ, TX, TY, TZ, CAMERA_ID, NAME")
+        f.write("#   POINTS2D[] as (X, Y, POINT3D_ID)")
+        f.write(f"# Number of cameras: {num_imgs}, mean observations per image: {mean_obs_per_img}\n")
```

Comment:
small change; based on the COLMAP format, I think this line should read `"Number of images: "`...

Reply:
Good catch, I just fixed that